include/openthread/mesh_diag.h

```diff
@@ -91,7 +91,7 @@ typedef struct otMeshDiagChildIterator otMeshDiagChildIterator;
 #define OT_MESH_DIAG_VERSION_UNKNOWN 0xffff
 
 /**
- * Represents information about a router in Thread mesh.
+ * Represents information about a router in Thread mesh discovered using `otMeshDiagDiscoverTopology()`
```

Comment:
I may have missed this one previously, missing period at end

Reply:
Fixed in new push. Thanks.